strapi/strapi
# Content type attribute is ignored if attribute name is repeated but c…

…apitalised



### What does it do? 

fixes #14012 

Here we're lower-casing all existing fields names and upcoming value by using `toLower() ` method imported from `Lodash` when we are creating or adding new content-type attribute

### Why is it needed?
we need this changes for they're going to help validation prevent user from not saving repeated names but capitalized
### How to test it?
- make sure that you are running the admin server cd ./packages/core/admin
- log to strapi dashboard 
- Go to the` plugins` section, then click on `Content-Type Builder`
- Pick a random Content-Type e.g. Category, trying to repeat used name fields by capitalized it

https://www.loom.com/share/93a0cee1af7b422db3f12954efd6a620
